assets/files.wxs

```diff
@@ -1964,6 +1964,11 @@
             <File Id="fil6B026F6D5210498B99E5F14386F4F90B" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\Microsoft.PowerShell.ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.Error occurred in Describe block
```powershell
Exception calling "GetHostEntry" with "1" argument(s): "nodename nor servname provided, or not known"
At /Users/runner/runners/2.163.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1:15 char:9
```